pkg/controllers/service/service_lb_controller.go

```diff
@@ -55,24 +61,31 @@ func (r *ServiceLbReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	}()
 
 	if err := r.Client.Get(ctx, req.NamespacedName, service); err != nil {
+		if client.IgnoreNotFound(err) == nil {
+			log.Info("Not found LB service", "req", req.NamespacedName)
+			return ResultNormal, client.IgnoreNotFound(err)
+		}
 		log.Error(err, "Failed to fetch LB service", "req", req.NamespacedName)
-		return ResultNormal, client.IgnoreNotFound(err)
+		return common.ResultRequeue, err
 	}
 
 	if service.Spec.Type == v1.ServiceTypeLoadBalancer {
-		log.Info("Reconciling LB service", "LBService", req.NamespacedName)
+		log.Info("Reconciling LB service", "LBService", req.NamespacedName, "version", service.ResourceVersion, "status", service.Status)
```

Comment:
put version and status in debug log level

Reply:
Done